x-pack/metricbeat/module/openai/usage/_meta/data.json

```diff
@@ -0,0 +1,38 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "openai.usage",
+        "duration": 115000,
+        "module": "openai"
+    },
+    "metricset": {
+        "name": "usage",
+        "period": 10000
+    },
+    "openai": {
+        "usage": {
+            "data": {
+                "aggregation_timestamp": "2024-11-04T05:01:00Z",
+                "api_key_id": null,
+                "api_key_name": null,
+                "api_key_redacted": null,
+                "api_key_type": null,
+                "email": null,
+                "n_cached_context_tokens_total": 0,
```

Comment:
@shmsr , @ishleenk17 - Are we okay with the field names? IMO, Updating the field names something similar to 
`n_context_tokens_total -> tokens.total or tokens_total` in the context of usability. But still it is also good to keep as it is to relate the ES fields with the openai API metrics. WDYT?

Reply:
Yeah, we should keep fields more readable and inline with other LLM Integrations

Reply:
I was thinking to keep it like this here and change the names in ingest pipelines in integrations. What do you think?

Reply:
If we want to have a parity between beats and Integrations, then ideally we should have similar field names

Reply:
But we don't want beats to be used right. So felt that ingest pipelines in integrations are also a good idea. But yes, no strong opinion. Doesn't matter much.

Reply:
Ideally we try to keep the fields in beats module and Integrtaions same. 
Here if particularly we are sure we are never going to use beats module. It should be ok.
@lalit-satapathy : WDYT ?

Reply:
Then if we are following that for other modules, let's do it here. I will rename the fields.

Reply:
Are we doing the name change of the fields ?
Then this sample json should also be updated along with the field name change

Reply:
I have changed the names. Based on fields names from Azure OpenAI. Although only a few fields were similar. But yes, I have changed the field names.